kfp_notebook/pipeline/_notebook_op.py

```diff
@@ -87,33 +89,37 @@ def __init__(self,
                 NOTE: Images being pulled must have python3 available on PATH and cURL utility
             """
 
+            argument_list.append('mkdir -p ./%s && cd ./%s && '
+                                 'curl -H "Cache-Control: no-cache" -L %s --output bootstrapper.py && '
+                                 'curl -H "Cache-Control: no-cache" -L %s --output requirements-elyra.txt && '
+                                 'python -m pip install packaging && '
+                                 'python -m pip freeze > requirements-current.txt && '
+                                 'python bootstrapper.py '
+                                 '--cos-endpoint %s '
+                                 '--cos-bucket %s '
+                                 '--cos-directory "%s" '
+                                 '--cos-dependencies-archive "%s" '
+                                 '--notebook "%s" ' % (
+                                     self.container_work_dir,
+                                     self.container_work_dir,
+                                     self.bootstrap_script_url,
+                                     self.requirements_url,
+                                     self.cos_endpoint,
+                                     self.cos_bucket,
+                                     self.cos_directory,
+                                     self.cos_dependencies_archive,
+                                     self.notebook
+                                     )
+                                 )
```

Comment:
Sorry for the late comment, but it would make things much easier to read here if a format construct was used...
```python
argument_list.append('mkdir -p ./{container_work_dir} && cd ./{container_work_dir} && '
                                 'curl -H "Cache-Control: no-cache" -L {bootstrap_script_url} --output bootstrapper.py && '
                                 'curl -H "Cache-Control: no-cache" -L {requirements_url} --output requirements-elyra.txt && '
                                 'python -m pip install packaging && '
                                 'python -m pip freeze > requirements-current.txt && '
                                 'python bootstrapper.py '
                                 '--cos-endpoint {cos_endpoint} '
                                 '--cos-bucket {cos_bucket} '
                                 '--cos-directory "{cos_directory}" '
                                 '--cos-dependencies-archive "{cos_dependencies_archive}" '
                                 '--notebook "{notebook}" '.format(
                                     container_work_dir=self.container_work_dir,
                                     bootstrap_script_url=self.bootstrap_script_url,
                                     requirements_url=self.requirements_url,
                                     cos_endpoint=self.cos_endpoint,
                                     cos_bucket=self.cos_bucket,
                                     cos_directory=self.cos_directory,
                                     cos_dependencies_archive=self.cos_dependencies_archive,
                                     notebook=self.notebook
                                     )
                                 )
```
Note: I didn't confirm that the above parses.  You could also use shorter tag names.  This prevents having to could '%s' entries to determine if the parameters are aligned to their placeholders.